src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510057>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510058>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510060>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510061>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510062>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510064>))
```

Comment:
>A mixture of gases and particles which is produced from the combustion of petroleum fuels in vehicles.

This should not be directly under `material entity`, but should be under `environmental material`

The definition should be "emissions from petroleum combustion" 

The definition should then read: "A material entity which is composed primarily of gases and particles produced by the combustion of petroleum fuels"

This should be axiomatised better. For example, stating this is `formed as a result of` some `combustion process` and `occurs in` some `motor` or similar

Reply:
The parent of 'emissions from petroleum combustion' is now 'environmental material'. The definition has been adapted as well, and an axiom for 'produced_by' some 'combustion process' has been included. We did not find a suitable example to model the occurs in relationship after.

Reply:
Thanks @laurenechan - will examine the latest files